BlueprintUICommonControls/Sources/AttributedLabel.swift

```diff
@@ -217,9 +217,14 @@ extension AttributedLabel {
             activeLinkAttributes = model.activeLinkAttributes
             linkDetectionTypes = model.linkDetectionTypes ?? []
 
-            attributedText = model
-                .attributedText
-                .normalizingForView(with: model.numberOfLines)
+            if isMeasuring {
+                attributedText = model
```

Comment:
For any other readers, I'll summarize `normalizingForView(numberOfLines:)`:

- Replace link attributes with custom link attribute to fully control link styling
- Update truncation mode to sensible defaults when bad values are provided. These updates are purported to not impact sizing.

I'm wondering if the first point can change sizing, especially when the default `UILabel` styling for links varies greatly from ours, but if we have snapshot / unit test coverage here that remained untouched by these changes, I would imagine we have little to worry about.